_includes/apply.html

```diff
@@ -1,4 +1,4 @@
-<section class="page-section">
+<section class="page-section call-to-action">
```

Comment:
I intentionally took this off earlier today because we have some CSS that's targeting that class for the apply/donate section at the top of the home page, and it doesn't play well with this page:

![Screen Shot 2020-04-05 at 10 07 39 PM](https://user-images.githubusercontent.com/3165665/78524853-13d2dd80-778a-11ea-84df-f3ef47ec3cf6.png)

Please back this out for now, and I/we can maybe take a crack at fixing the spacing between the logo and the title a little more surgically.